lib/partition_healing/discover_provider_healer.js

```diff
@@ -63,6 +63,7 @@ require('util').inherits(DiscoverProviderHealer, Healer);
  */
 DiscoverProviderHealer.prototype.heal = function heal(callback) {
     var self = this;
+    self.ringpop.stat('increment', 'heal.triggered.discover_provider');
```

Comment:
I think that's fine, but let's keep the stats the same by either changing go to this style, or changing node to go's style/